UPGRADE-4.2.md

```diff
@@ -11,6 +11,22 @@ Config
 
  * Deprecated constructing a `TreeBuilder` without passing root node information.
 
+Console
+-------
+
+ * Deprecated passing a command as a string to `ProcessHelper::run()`,
+   pass it the command as an array of arguments instead.
```

Comment:
pass the command

Reply:
I would also add a word about the new `withShell()` method (as passing an array of arguments is not always possible, perhaps mention the cases where using `withShell()` is "valid"?).